### PR TITLE
Cold flow attempt code changes

### PR DESCRIPTION
**Pull request by sebastiengrd**

### CODE REVIEW

File: src/IO/tcp/SocketServer.cpp [hunk 1]
```diff
@@ -79,10 +79,14 @@ void SocketServer::sendingLoop()
...
-            {
-                client->send(data);
+            
+            if (clients.size() > 0 && clients[clients.size() - 1] != nullptr) {
+                clients[clients.size() - 1]->send(data);
             }
+            //for (auto &client : clients)
+            //{
+            //    client->send(data);
+            //}
```
Comment (ajayyy) [inline src/IO/tcp/SocketServer.cpp:89]: We can undo this change, but other than that, I think it is good to merge

Comment (jonapap) [reply src/IO/tcp/SocketServer.cpp:89]: Yeah, we need to debug and rework that part of the code.

**Review by jonapap — DISMISSED** [review]

**Comment by ajayyy** [comment]
To fix the formatting, you can use ctrl + shift _+ F in vscode

File: projects/HotFire/config/UOStateMachine.cpp [hunk 2]
```diff
@@ -201,7 +201,7 @@ STATE_DEFINE(UOStateMachine, WaitForInit, UOSMData)
 
     if (interfaceData->isInitialized())
     {
-        InternalEvent(ST_WAIT_FOR_FILLING);
+        InternalEvent(ST_SERVO_CONTROL);
```
Comment (ajayyy) [inline projects/HotFire/config/UOStateMachine.cpp:204]: Need to revert this change

**Review by ajayyy — APPROVED** [review]

**Review by ajayyy — APPROVED** [review]

**Review by jonapap — APPROVED** [review]
